docs/docs/6.x/installation.md

```diff
@@ -5,10 +5,13 @@
 composer require santigarcor/laratrust
 ```
 
-2. Publish all the configuration files:
+2. Publish the configuration file:
+
 ```bash
-php artisan vendor:publish --tag="laratrust"
+    php artisan vendor:publish --provider="Laratrust\LaratrustServiceProvider" --tag="config"   
```

Comment:
I would prefer to leave the first one `php artisan vendor:publish --tag="laratrust"`